pkgs/tools/networking/updog/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "updog";
+  version = "1.4";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "7n/ddjF6eJklo+T79+/zBxSHryebc2W9gxwxsb2BbF4=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ colorama flask flask-httpauth werkzeug pyopenssl ];
```

it's starting to wrap around
```suggestion
  propagatedBuildInputs = with python3Packages; [
    colorama
    flask
    flask-httpauth
    werkzeug
    pyopenssl
  ];
```